lib/lbt/bundle/Builder.js

```diff
@@ -220,6 +228,8 @@ class BundleBuilder {
 			return this.writeBundleInfos([section]);
 		case SectionType.Require:
 			return this.writeRequires(section);
+		case SectionType.DepCache:
```

Comment:
The AutoSplitter also needs to know about the DepCache SectionType. Otherwise it fails to calculate the correct bundle size in order to decide which parts of the bundle to put into separate files. The goal of the AutoSplitter is to split bundle content evenly across multiple parts (option: numberOfParts > 1).

It also seems that this has been missed to be implemented for the BundleInfo mode.

Reply:
Thanks!